base/dict.jl

```diff
@@ -189,6 +189,16 @@ function grow_to!{K,V}(dest::Associative{K,V}, itr, st)
     return dest
 end
 
+
+"""
+    similar(d::Dict, [element_type=Pair{K,V}])
+
+Create an empty dictionary with key type `K` and value type `V`. The second element
```

Comment:
> Is Dict{eltype(d)}() longer than similar(d), or similar(d, eltype(e)), except perhaps that the constructor does not exist? I have asked this question on the mailing list just now.

The `Dict{eltype(d)}()` constructor cannot exist since `Dict` needs two parameters.

> To summarize, I will only leave the first two lines and drop the example. Would that be OK?

I would say only the first sentence is needed (after removal of mentions of `K` and `V`). Examples wouldn't hurt, though, if you want to add a pair of them.